src/backend/sev/snp/firmware/mod.rs

```diff
@@ -134,7 +150,7 @@ pub struct TcbVersion {
     /// Current PSP OS version.
     /// SVN of PSP operating system.
     pub tee: u8,
-    _reserved: [u8; 4],
+    pub _reserved: [u8; 4],
```

Comment:
We could avoid needing to do this by introducing a builder/constructor of some sort, but not sure if that's really necessary, since this struct is meant to be filled by C